.changeset/olive-fans-invite.md

```diff
@@ -2,7 +2,9 @@
 "@comet/admin": minor
 ---
 
-Add new `Button` component to use in favor of MUI's `Button` component
+Add new `Button` with optional responsive behavior to use in favor of MUI's `Button`
+
+When setting the `responsive` prop, the button will only show the icon on mobile and show the text content in a tooltip.
```

Comment:
Is there a reason why this shouldn't be the default behavior?

Reply:
- Backwards compatibility with MUIs `Button`
- Often used with no content next to it, where there is enough space for the full button's text, e.g. a single Button in `DialogActions` or something like a "Submit" button at the bottom of a for/page. 

We could discuss with UX if this makes sense for V8.

Reply:
Okay, let's keep it optional for now. If we find that it's used all the time, we could instead implement a `disableResponsiveBehavior` prop in the future.